.github/workflows/move-inprogress-to-review.yml

```diff
@@ -15,4 +15,4 @@ jobs:
           project-url: https://github.com/orgs/cern-sis/projects/1
           column-name: "Review"
           label-name: "ready-to-review"
-          colums-to-ignore: "On hold, Todo, QA, Ready to Deploy, PROD, Done"
\ No newline at end of file
+         
```

Comment:
could you please also add a new line at the end of the file? You can configure your editor to do that by default.